Sources/Moya/Response.swift

```diff
@@ -38,14 +42,17 @@ public extension Response {
         return self
     }
 
+    /// Filters out responses of a specific status code, generating errors when others are encountered.
```

Comment:
Yeah, I was just going off of the preexisting documentation for consistency. But I will change that as well.